packages/client/src/components/atoms/AddToCalendar/AddToCalendar.tsx

```diff
@@ -93,27 +97,28 @@ const AddToCalendar: React.FC = () => {
     dispatch(sendICSFile({ icsFile: icsFile, email, secretKey }));
   };
 
-  return bookedSlots.length ? (
+  // Don't render the component if there are no booked slots to save to calendar
+  if (!bookedSlots.length) return null;
+
+  return (
     <>
       <Button
-        className=" text-white bg-cyan-700 m-auto justify-center
-        position-relative
-         active:bg-cyan-800 md:w-auto  invisible md:visible "
+        className="hidden bg-cyan-700 text-white md:block active:bg-cyan-800"
```

Comment:
'hidden' is `display: none`, while 'invisible' is `display: hidden` (or something like that). The difference:

https://www.edureka.co/community/62212/explain-difference-between-visibility-hidden-display-none#:~:text=display%3Anone%20means%20that%20the,t%20seen%20on%20the%20page.

You can see the effect in dev tools (on small screen) of the original branch: the button isn't shown, but is still taking up space, while in this branch, the button is removed from the flow.